FIRO_TSEnsembles/src/main/java/hec/ensemble/Ensemble.java

```diff
@@ -124,7 +124,7 @@ public float[][] multiComputeForTracesAcrossTime(MultiComputable cmd){
         ((Configurable)cmd).configure(_configuration);
       }
       int size= values.length;
-      int size2 = cmd.Statistics().length;
+      int size2 = cmd.StatisticsLabel().split("\\|").length;
```

Comment:
Consider moving that *magic* label parsing to find the dimension size into a higher base class or interface.
```java
int MultiComputagle.getCount(){
return cmd.StatisticsLabel().split("\\|").length;
// or perhaps something better.  Perhaps, when computables are defined , have a count variable defined?
}
```

Reply:
Added a method in the MultiComputable interface to get the numbers of stats computed.